tidy3d/components/scene.py

```diff
@@ -1209,8 +1269,15 @@ def _get_structure_eps_plot_params(
             # 2d material
             plot_params = plot_params.copy(update={"edgecolor": "k", "linewidth": 1})
         else:
-            # regular medium
-            eps_medium = medium.eps_model(frequency=freq).real
+            if component and isinstance(medium, (AnisotropicMedium, FullyAnisotropicMedium)):
```

Comment:
this seems like duplicated code from above. we should refactor to put it somewhere it can be re-used, potentially we can just put it into `eps_comp` as I mentioned in the earlier comment.